examples/scheduler/src/test/kotlin/cnt/scheduler/AnnotationBasedSchedulerIntTest.kt

```diff
@@ -0,0 +1,38 @@
+package cnt.scheduler
+
+import io.mockk.verify
+import io.quarkiverse.test.junit.mockk.InjectSpy
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.QuarkusTestProfile
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.Test
+
+/**
+ * This Integration test is not really useful
+ * as this implementation heavily depends on the "out of the box" quarkus scheduler implementation itself.
+ * It is also not recommended to test something where you are not able to change its implementation.
+ */
+@QuarkusTest
+@TestProfile(EnabledAnnotationBasedScheduler::class)
+class AnnotationBasedSchedulerIntTest {
+
+    @InjectSpy
+    private lateinit var sendBookNewsletter: SendBookNewsletter
+
+    @Test
+    fun `enabled scheduler - runs three times according configuration`() {
+        Thread.sleep(100)
+        verify(atLeast = 3) { sendBookNewsletter.invoke() }
```

Comment:
At least called once should be enough. We don't want to measure the exact call amount, and with such a small wait a badly timed GC cycle can break your test.

Reply:
Good idea